local-process-quickstart.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+
+# Please install the below pre-requisites if using this on your local machine, or alternately, you can just use azure bash cloudshell for a seamless experience.
+#1. az
+#2. kubectl
+#3. curl
+#4. gunzip
+#5. tar
+#6. perl
+#7. cut
+
+INGRESSNAME=bikesharing-traefik
+PIPNAME=BikeSharingPip
+HELMDIR=/var/tmp/helm_azds
+HELMROOT=${HELMDIR}/darwin-amd64
+
+helpFunction()
+{
+   echo ""
+   echo "Usage: $1 -g ResourceGroupName -n AKSName -r DevSpacesRepoRoot"
+   echo -e "\t-g Name of resource group of AKS Cluster"
+   echo -e "\t-n Name of AKS Cluster"
+   echo -e "\t-r Path to Root of dev spaces repo"
+   echo -e "\t-c Cleanup"
+   exit 1 # Exit script after printing help
+}
+
+installHelmFunction()
+{
+   mkdir $HELMDIR
+   if [[ "$OSTYPE" == "linux-gnu"* ]]; then
```

Comment:
>if [[ "$OSTYPE" == "linux-gnu"* ]]; then [](start = 3, length = 40)

I'm a bit reluctant to let the users spend time installing everything on Linux, as we do not support it yet in VS Code/VS.
I'm fine letting the Linux logic inside the script, but should we also have a specific message at the beginning of the script letting Linux users know that, unfortunately, as of today Linux isn't supported?

Reply:
So the linux part of the script is to make it support on bash cloudshell. 
I think this may not be the right place to tell users that connect does not work on linux today. We should bubble that up in the documentation instead.